sql-plugin/src/main/311until330-all/scala/com/nvidia/spark/rapids/shims/GpuTypeShims.scala

```diff
@@ -82,15 +83,55 @@ object GpuTypeShims {
     throw new RuntimeException(s"Not support type $dt.")
 
   /**
-   * Spark supports interval type from 320; Spark supports to/from Parquet from 330,
-   * And lots of interval operators from 330, so just return false for Spark330-
+   * Alias supported types for this shim
    */
-  def isDayTimeIntervalType(dt: DataType) : Boolean = false
+  def supportedTypesForAlias: TypeSig = TypeSig.none
 
   /**
-   * Spark supports interval type from 320; Spark supports to/from Parquet from 330,
-   * And lots of interval operators from 330, so just return false for Spark330-
+   * Abs supported types for this shim
    */
-  def isYearMonthIntervalType(dt: DataType) : Boolean = false
+  def supportedTypesForAbs: TypeSig = TypeSig.none
```

Comment:
`supportedTypesForAbs` sounds like a method that should return all the types that are supported for the `Abs` expression, but this returns `TypeSig.none` which is very surprising.  The name should explain this is not the complete list, e.g.: "additionalSupportedTypes" or something similar, if we keep these separate methods (and I hope we don't).

Comment applies to all the other methods that are similarly named.